content/influxdb/cloud-dedicated/write-data/use-telegraf/configure/manual-config.md

```diff
@@ -0,0 +1,180 @@
+---
+
+Use the Telegraf `influxdb_v2` output plugin to collect and write metrics into
+an InfluxDB {{< current-version >}} bucket.
+This article describes how to enable the `influxdb_v2` output plugin in new and
+existing Telegraf configurations,
+then start Telegraf using the custom configuration file.
+
+{{< youtube qFS2zANwIrc >}}
+
+{{% note %}}
+_View the [requirements](/influxdb/cloud-dedicated/write-data/use-telegraf#requirements)
+for using Telegraf with InfluxDB {{< current-version >}}._
+{{% /note %}}
+
+<!-- TOC -->
+
+- [Configure Telegraf input and output plugins](#configure-telegraf-input-and-output-plugins)
+  - [Manually add Telegraf plugins](#manually-add-telegraf-plugins)
+  - [Enable and configure the InfluxDB v2 output plugin](#enable-and-configure-the-influxdb-v2-output-plugin)
+      - [urls](#urls)
+      - [token](#token)
+        - [Avoid storing tokens in telegraf.conf](#avoid-storing-tokens-in-telegrafconf)
+      - [organization](#organization)
+      - [bucket](#bucket)
+    - [Example influxdb_v2 configuration](#example-influxdb_v2-configuration)
+      - [Write to InfluxDB v1.x and v2.6](#write-to-influxdb-v1x-and-v26)
+- [Start Telegraf](#start-telegraf)
+
+<!-- /TOC -->
+
+## Configure Telegraf input and output plugins
+
+Configure Telegraf input and output plugins in the Telegraf configuration file (typically named `telegraf.conf`).
+Input plugins collect metrics.
+Output plugins define destinations where metrics are sent.
+
+_See [Telegraf plugins](/{{< latest "telegraf" >}}/plugins/) for a complete list of available plugins._
+
+### Manually add Telegraf plugins
+
+To manually add any of the available [Telegraf plugins](/{{< latest "telegraf" >}}/plugins/), follow the steps below.
+
+1.  Find the plugin you want to enable from the complete list of available
+    [Telegraf plugins](/{{< latest "telegraf" >}}/plugins/).
+2.  Click **View** to the right of the plugin name to open the plugin page on GitHub.
+    For example, view the [MQTT plugin GitHub page](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/mqtt_consumer/README.md).
+3.  Copy and paste the example configuration into your Telegraf configuration file
+    (typically named `telegraf.conf`).
+
+### Enable and configure the InfluxDB v2 output plugin
+
+To send data to an InfluxDB {{< current-version >}} instance, enable the
+[`influxdb_v2` output plugin](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/influxdb_v2/README.md)
+in the `telegraf.conf`.
+
+To find an example InfluxDB v2 output plugin configuration in the UI:
+
+1. In the navigation menu on the left, select **Load Data** > **Telegraf**.
+
+    {{< nav-icon "load data" >}}
+
+2. Click **InfluxDB Output Plugin**.
+3. Click **Copy to Clipboard** to copy the example configuration or **Download Config** to save a copy.
+4. Paste the example configuration into your `telegraf.conf` and specify the options below.
```

Comment:
There's no UI with dedicated, so we should just show an example config here in the docs.

~~~suggestion
```toml
[[outputs.influxdb_v2]]
  urls = ["https://cluster-id.influxdb.io"]
  token = "$INFLUX_TOKEN"
  organization = ""
  bucket = "$INFLUX_DATABASE"
```
~~~

Reply:
> There's no UI with dedicated, so we should just show an example config here in the docs.

Oops, thought I deleted all the UI stuff.